components/classroom/index.html

```diff
@@ -1,50 +1,67 @@
+        }
+    </style>
 </head>
-    <body>
-        <div class="header">
-            <h1>TPEN Classroom Interface</h1>
-            <div class="hamburger-menu" onclick="toggleMenu()">
-                &#9776 <!-- Hamburger icon -->
-            </div>
+
+<body>
+    <div class="header">
+        <h1>TPEN Classroom Interface</h1>
+        <div class="hamburger-menu" onclick="toggleMenu()">
+            &#9776 <!-- Hamburger icon -->
         </div>
-            <div class ="navbar" id="navbar">
-            <ul>
-                <li><a href="./index.html">Home</a></li>
-                <li><a href="./index.html">Annotations</a></li>
-                <li><a href="./index.html">Groups</a></li>
-                <li><a href="./index.html">Contact</a></li>
-            </ul>
-            </div>
+    </div>
+    <div class="navbar" id="navbar">
+        <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./index.html">Annotations</a></li>
+            <li><a href="./index.html">Groups</a></li>
+            <li><a href="./index.html">Contact</a></li>
+        </ul>
+    </div>
 
-            <div class="page-container">
-                <h1>Welcome!</h1>
-                <div class="input-container">
-                    <div class="label-container">
-                        <form id="projectForm" class="input-form">
-                            <label for="projectId">Project ID:</label>
-                            <input id="projectId" name="projectId" type="text" required>
-                            <button class="submit-button" type="submit">Submit</button>
-                        </form>
-                        <!--<form class="input-form">
+    <div class="page-container">
+        <h1>Welcome!</h1>
+        <div class="input-container">
+            <div class="label-container">
+                <form id="projectForm" class="input-form">
+                    <label for="projectId">Project ID:</label>
+                    <input id="projectId" maxlength="24" name="projectId" type="text" required>
+                    <button class="submit-button" type="submit">Submit</button>
+                </form>
+                <tpen-projects-list id="clickList"></tpen-projects-list>
+                <!--<form class="input-form">
                             <label for="password">Password:</label>
                             <input id="password" name="password" type="password">
                         </form>-->
-                        <p id="error-msg" style="color: red; display: none;">Project not Found</p>
-                    </div>
-                </div>
+                <p id="msg"></p>
             </div>
+        </div>
+    </div>
 
-        <footer>
-            <p>TPEN Classroom Interface</p>
-        </footer>
+    <footer>
+        <p>TPEN Classroom Interface</p>
+    </footer>
+</body>
 
-        <script src="./menutoggle.js"></script>
-        <script type="module" src="./fetchData.mjs"></script>
```

Comment:
This is in the wrong place. The type=module already defers its loading, so there is no good reason to drop scripts down here.